core.js

```diff
@@ -238,11 +238,17 @@ function create_patch_request(ch, pull) {
               'title':          pull.title,
               'created':        new Date(pull.created_at),
               'merged':         merge_date,
+              'score':          0,
+              'rating':         0,
               'lines_inserted': lines_inserted,
               'lines_removed':  lines_removed,
               'files_changed':  files_changed
             }
 
+            // Update score if formulae is valid
+            if (exports.eval_formulae(ch.formulae))
+              newpr['score'] = exports.eval_formulae(ch.formulae, newpr)
```

Comment:
This is related with my question here https://github.com/rosedu/challenge/pull/43/files#r22825835